dw-date-range-picker.js

```diff
@@ -543,6 +441,7 @@ export class DwDateRangePicker extends DwCompositeDialog {
 
     this._instance.on('preselect', this._onPreselect);
     this._instance.on('selected', this._onSelected);
+    this._instance.on('preselect', this._onPreselect);
```

Comment:
@ujjval-kukadiya 'preselect' event is listened once on line 442. Why its needed 2nd time?